challenge_5/java/jdfurlan/src/FindTheDifference.java

```diff
@@ -0,0 +1,21 @@
+
+public class FindTheDifference {
+
+	public static void main(String[] args) {
+		String s = "abcd";
+		String t = "dbeca";
+
+		boolean[] b = new boolean[128];
+		for (int i = 0; i < s.length(); i++) {
+			b[s.charAt(i)] = true;
```

Comment:
Yes it does. eg. `String s = "apple"; String t = "plepva";` output is `v found at index: 4` the reason being every character, if found, is stored in its char value index in a boolean array. regardless of how many are present in the string, it will will see `true` for that char and know that it was present in the original string.